src/current/molt/migration-overview.md

```diff
@@ -0,0 +1,136 @@
+|---------------------------------------------|------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| `--mode data-load`                          | Performs one-time load of source data into CockroachDB                       | Testing, migrations with planned downtime, [phased migrations]({% link molt/migrate-in-phases.md %}) |
+| `--mode data-load-and-replication`          | Loads source data and starts continuous replication from the source database | [Migrations with minimal downtime]({% link molt/migrate-to-cockroachdb.md %})                        |
+| `--mode replication-only`                   | Starts replication from a previously loaded source                           | [Phased migrations]({% link molt/migrate-in-phases.md %}), post-load sync                            |
+| `--mode failback`                           | Replicates changes on CockroachDB back to the original source                | [Rollback scenarios]({% link molt/migrate-failback.md %})                                            |
+| `--mode export-only` / `--mode import-only` | Separates data export and import phases                                      | Large-scale migrations, custom storage pipelines                                                     |
+| `--direct-copy`                             | Loads data without intermediate storage using `COPY FROM`                    | Local testing, limited infra environments                                                            |
+
+## Migrations with minimal downtime
+
+MOLT simplifies and streamlines the following migration patterns that use a replication stream to minimize downtime. Rather than load all data into CockroachDB during a planned downtime window, you perform an initial data load and continuously replicate any subsequent changes to CockroachDB. Writes are only briefly paused to allow replication to drain before final cutover.
+
+### Full migration with minimal downtime
+
+Run MOLT Fetch in `data-load-and-replication` mode to load the initial source data into CockroachDB. Continuous replication starts automatically after the initial load. When ready, pause application traffic to allow replication to drain, validate data consistency with MOLT Verify, then cut over to CockroachDB. For example steps, refer to [Migrate to CockroachDB]({% link molt/migrate-to-cockroachdb.md %}).
+
+### Phased migration with minimal downtime
+
+Run MOLT Fetch in `data-load` mode to incrementally load and validate data in batches. After loading the initial source data, switch to `replication-only` mode to sync ongoing changes. When ready, pause application traffic to allow replication to drain, validate again with MOLT Verify, then cut over to CockroachDB. For example steps, refer to [Migrate in Phases]({% link molt/migrate-in-phases.md %}).
```

Comment:
I'm having a hard time understanding this block here with the Migration with mniimal downtime above. Maybe its worth explaining more about the batches? Unless that is covered in the linked page. But at least from directly reading this, its hard to see a difference.

Reply:
Okay sounds good. I think maybe Phase migration here still doesn't quite make sense to me. Since this is still a full migration, just you perform some schema changes between potentailly.

Reply:
Yeah, this section is totally gone in the new version!